src/ray/raylet/worker_pool.cc

```diff
@@ -197,24 +186,19 @@ void WorkerPool::update_worker_startup_token_counter() {
   worker_startup_token_counter_ += 1;
 }
 
-void WorkerPool::AddStartingWorkerProcess(
+void WorkerPool::AddWorkerProcess(
```

Comment:
What's the motivation for the name change?

Reply:
Before current PR, we will delete the process when the worker finishes registration. 
In current PR, I change this logic. The process will be stored all the way before the worker exits.